metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -1203,6 +1210,23 @@ def gen_schema_metadata(
     def get_report(self) -> BigQueryV2Report:
         return self.report
 
+    def get_views_for_dataset(
+        self, conn: bigquery.Client, project_id: str, dataset_name: str
+    ) -> Iterable[BigqueryView]:
+
+        with PerfTimer() as timer:
+            table_items = self.get_core_table_details(conn, dataset_name, project_id)
+            yield from BigQueryDataDictionary.get_views_for_dataset(
+                conn,
+                project_id,
+                dataset_name,
+                has_data_read=self.config.is_profiling_enabled(),
+                table_items=table_items,
+            )
+        self.report.metadata_extraction_sec[f"{project_id}.{dataset_name}"] = round(
```

Comment:
This would overwrite the table extraction timing. 
```suggestion
        self.report.metadata_extraction_sec[f"{project_id}.{dataset_name}"] += round(
```